components/common/SelectDropdown.tsx

```diff
@@ -81,11 +81,15 @@ function getSelectStyles<Option extends SelectDropdownOption>(
       const { indent } = data;
       const ret = {
         ...provided,
-        color: theme.themeColors.black,
+        color: isSelected
+          ? theme.themeColors.white
+          : isFocused
+          ? theme.themeColors.black
+          : theme.themeColors.black,
```

Comment:
Is the second ternary operator needed? Or should those colours be different depending on focus?

```suggestion
        color: isSelected
          ? theme.themeColors.white
          : theme.themeColors.black
```